mod-search/src/main/resources/karate-config.js

```diff
@@ -0,0 +1,72 @@
+function fn() {
+  karate.configure('logPrettyRequest', true);
+  karate.configure('logPrettyResponse', true);
+
+  var env = karate.env;
+
+  // specify runId property for tenant postfix to avoid close connection issues
+  // once we run tests again
+  var runId = karate.properties['runId'];
+
+  var config = {
+    baseUrl: 'https://falcon-okapi.ci.folio.org',
+    admin: {tenant: 'diku', name: 'diku_admin', password: 'admin'},
```

Comment:
Hardcoded password?

Reply:
Yes, all modules has hardcoded passwords. No way to inject variables exists now.

Reply:
> Is there a way to share karate-config.js with other modules?

The current approach is that each module is isolated and defines it's own `karate-config.js`